front/app/modules/commercial/insights/admin/containers/Insights/Details/Network/index.tsx

```diff
@@ -295,22 +432,54 @@ const Network = ({
             }
           />
         </SectionTitle>
+        {hiddenNodes.length > 0 && (
```

Comment:
This one cal also perhaps be taken out as a separate component, if that makes sense?